packages/debian/entrypoint.sh

```diff
@@ -35,6 +35,19 @@ rpm2deb() {
     chown "${TARGET_UID}:${TARGET_GID}" *
 }
 
+buildrepo(){
+    set -x
+    mkdir -p /var/www/debian/conf
+    mkdir -p /var/www/debian/incoming
+    # Mount `distributions` file in  /var/www/debian/conf
+    pushd /var/www/debian
+    reprepro -b . export
+    # We download packages that we want
+    reprepro -vb . includedeb <codename> <Path to the .deb>
+    # A folder `pool` is created whith all .deb insinde
+
+
+## TODO : Handle the GPG key or find a way to do without.
```

Comment:
To be discussed if we are OK "with a way without", I guess we want signed stuff for security purpose.
To be discussed with the team.